### PR TITLE
Viite 2366 hotfix to creation/handling of dual roadway points

### PR DESCRIPTION
**Pull request by andre-policarpo**
[Tests **OK**]: http://livibuild04.vally.local/jenkins/job/viite/job/Run-Tests/283/consoleFull

### CODE REVIEW

File: digiroad2-viite/src/test/scala/fi/liikennevirasto/viite/RoadAddressServiceSpec.scala [hunk 1]
```diff
@@ -838,6 +837,127 @@ class RoadAddressServiceSpec extends FunSuite with Matchers{
...
+      val roadwayNumber1 = Sequences.nextRoadwayNumber
+      val roadwayNumber2 = Sequences.nextRoadwayNumber
+      val roadwayNumber3 = Sequences.nextRoadwayNumber
+
+      val projectId = Sequences.nextViiteProjectId
+      val rap =  dummyProject(projectId, ProjectState.Incomplete, Seq(), None).copy(startDate = DateTime.parse("2019-10-10"))
+      val link1 = dummyProjectLink(99, 1, Track.Combined, Discontinuity.EndOfRoad, 0 , 5, Some(DateTime.now()), None, 12345, 0, 5, SideCode.TowardsDigitizing, LinkStatus.UnChanged, projectId, RoadType.PublicRoad, geom1, roadwayNumber2)
+      val link2 = dummyProjectLink(99, 2, Track.Combined, Discontinuity.EndOfRoad, 0 , 15, Some(DateTime.now()), None, 12346, 0, 15, SideCode.TowardsDigitizing, LinkStatus.Transfer, projectId, RoadType.FerryRoad, geom2, roadwayNumber3)
+
+//      val link1 = dummyProjectLink(99, 2, Track.Combined, Discontinuity.Continuous, 0 , 5, Some(DateTime.now()), None, 12345, 0, 5, SideCode.TowardsDigitizing, LinkStatus.Terminated, projectId, RoadType.PublicRoad, geom1, roadwayNumber1)
```
Comment (Campanha) [inline digiroad2-viite/src/test/scala/fi/liikennevirasto/viite/RoadAddressServiceSpec.scala:853]: we should remove commented lines

Comment (andre-policarpo) [reply digiroad2-viite/src/test/scala/fi/liikennevirasto/viite/RoadAddressServiceSpec.scala]: done

File: digiroad2-viite/src/main/scala/fi/liikennevirasto/viite/dao/CalibrationPointDAO.scala [hunk 2]
```diff
@@ -109,14 +130,14 @@ object CalibrationPointDAO {
...
-         JOIN ROADWAY_POINT RP
-          ON RP.ID = CP.ROADWAY_POINT_ID
-         WHERE cp.roadway_point_id = $roadwayPointId
-     """.as[CalibrationPoint].firstOption
+  def fetchByRoadwayPointId(roadwayPointId: Long): Seq[CalibrationPoint] = {
+    val query = s"""
+        SELECT CP.ID, ROADWAY_POINT_ID, LINK_ID, ROADWAY_NUMBER, RP.ADDR_M, START_END, CP.TYPE, VALID_FROM, VALID_TO, CP.CREATED_BY, CP.CREATED_TIME
+        FROM CALIBRATION_POINT CP
+        JOIN ROADWAY_POINT RP ON RP.ID = CP.ROADWAY_POINT_ID
+          WHERE cp.roadway_point_id = $roadwayPointId
```
Comment (skosonen) [inline digiroad2-viite/src/main/scala/fi/liikennevirasto/viite/dao/CalibrationPointDAO.scala:138]: `AND cp.valid_to is null`

Comment (andre-policarpo) [reply digiroad2-viite/src/main/scala/fi/liikennevirasto/viite/dao/CalibrationPointDAO.scala]: done, this can be merged

**Pull request merged**